notebook/notebookapp.py

```diff
@@ -785,9 +790,20 @@ def _notebook_dir_changed(self, name, old, new):
         help="Reraise exceptions encountered loading server extensions?",
     )
 
+    iopub_msg_rate_limit = Float(None, config=True, allow_none=True, help="""(msg/sec)
```

Comment:
Probably use `0` for the no-limit indicator. This is what we do elsewhere, so we should be consistent.